make-release.sh

```diff
@@ -0,0 +1,127 @@
+#!/bin/bash
+# Release process automation script. 
+# Used to create branch/tag, update VERSION files 
+# and and trigger release by force pushing changes to the release branch 
+
+# set to 1 to actually trigger changes in the release branch
+TRIGGER_RELEASE=0 
+NOCOMMIT=0
+
+while [[ "$#" -gt 0 ]]; do
+  case $1 in
+    '-t'|'--trigger-release') TRIGGER_RELEASE=1; NOCOMMIT=0; shift 0;;
+    '-r'|'--repo') REPO="$2"; shift 1;;
+    '-v'|'--version') VERSION="$2"; shift 1;;
+    '-n'|'--no-commit') NOCOMMIT=1; TRIGGER_RELEASE=0; shift 0;;
+  esac
+  shift 1
+done
+
+bump_version () {
+  CURRENT_BRANCH=$(git rev-parse --abbrev-ref HEAD)
+
+  NEXTVERSION=$1
+  BUMP_BRANCH=$2
+
+  git checkout ${BUMP_BRANCH}
+
+  echo "Updating project version to ${NEXTVERSION}"
+  npm --no-git-tag-version version ${NEXTVERSION}
+
+  if [[ ${NOCOMMIT} -eq 0 ]]; then
+    COMMIT_MSG="[release] Bump to ${NEXTVERSION} in ${BUMP_BRANCH}"
+    git commit -s -m "${COMMIT_MSG}" VERSION
+    git pull origin "${BUMP_BRANCH}"
+
+    PUSH_TRY="$(git push origin "${BUMP_BRANCH}")"
+    # shellcheck disable=SC2181
+    if [[ $? -gt 0 ]] || [[ $PUSH_TRY == *"protected branch hook declined"* ]]; then
+    PR_BRANCH=pr-${BUMP_BRANCH}-to-${NEXTVERSION}
+      # create pull request for master branch, as branch is restricted
+      git branch "${PR_BRANCH}"
+      git checkout "${PR_BRANCH}"
+      git pull origin "${PR_BRANCH}"
+      git push origin "${PR_BRANCH}"
+      lastCommitComment="$(git log -1 --pretty=%B)"
+      hub pull-request -f -m "${lastCommitComment}
+${lastCommitComment}" -b "${BUMP_BRANCH}" -h "${PR_BRANCH}"
```

Comment:
What this command is supposed to do?

Reply:
hub is a command line tool used to create pull requests, so here a pull request will be created from "PR_BRANCH" to "BUMP_BRANCH"

Reply:
although I might fix this "duplicated" commit  message

Reply:
> although I might fix this "duplicated" commit message

Then missing \ in the previous line as well.